docs/docs/tutorials/spring-boot-integration.md

```diff
@@ -13,7 +13,7 @@ To use one of the Spring Boot starters, first import the corresponding dependenc
 ```xml
 <dependency>
     <groupId>dev.langchain4j</groupId>
-    <artifactId>langchain4j-open-ai-spring-boot-starter</artifactId>
+    <artifactId>langchain4j-spring-boot-starter</artifactId>

```

Comment:
It appears that the next documentation below this block primarily focuses on the OpenAI implementation and will not work with just `langchain4j-spring-boot-starter`:


Then, you can configure model parameters in the `application.properties` file as follows:
```
langchain4j.open-ai.chat-model.api-key=${OPENAI_API_KEY}
```

Reply:
@Wadimz is correct. I have updated the docu to make it more clear.